source/features/dim-bots.css

```diff
@@ -6,7 +6,8 @@
 /* Commit titles, dim */
 .rgh-dim-bots:not(.rgh-tagged, :hover) > *,
 /* PR row, dim */
-.rgh-dim-bots:not(.rgh-tagged, :hover) .Box-row--drag-hide {
+	.rgh-dim-bots:not(.rgh-tagged, :hover)
+	.Box-row--drag-hide {
```

Comment:
It looks like they treat the comments as part of the same line, so `.rgh-dim-bots` wrap it and indent it :(